patches/filesystem-chmod-0001.patch

```diff
@@ -0,0 +1,15 @@
+--- a/core/lib/Drupal/Core/File/FileSystem.php	2023-02-27 12:38:42.368159781 +0000
++++ b/core/lib/Drupal/Core/File/FileSystem.php	2023-02-27 12:38:20.107897836 +0000
+@@ -106,6 +106,12 @@
+       return TRUE;
+     }
+ 
++    // We may get forced ogu+rwx permissions from Azure Blob Storage, consider this a success.
```

Comment:
Looks ok but is this the only place to fix the issue? Would be better if it could be fixed somewhere that only affects images

Reply:
There's are methods use this, which check if the chmod was a success. We could have changed that to ignore the failure, but decided to go for this one to help prevent other errors we haven't found yet. This method will always return failure on Azure blob storage, so seemed fair enough.

The thing I was wondering is if it would have been better to check if the mode was correct, and alter the config for target mode.